lib/routes/bulk-get.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+var utils = require('../utils');
+
+module.exports = function (app) {
+  utils.requires(app, 'routes/db');
+
+  // Bulk docs operations
+  app.post('/:db/_bulk_get', utils.jsonParser, function (req, res, next) {
+
+    var opts = req.body || {};
+
+    if (Array.isArray(req.body)) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Request body must be a JSON object"
+      });
+    }
+    if (!req.body.docs) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Missing JSON list of 'docs'"
+      });
+    }
+
+    // these can be specified via query, e.g. ?revs=true
+    ['revs', 'attachments', 'atts_since'].forEach(function (param) {
```

Comment:
atts_since cannot be in query params since this option applies per document basis in payload. You unlikely (IRL) will have any two documents which shares the same revisions.